examples/routing_service/udp_socket_adapter_typed/RsSocketAdapter.xml

```diff
@@ -70,11 +69,61 @@
                             You could use Triangle or Circle if you also modify the other
                             references to Square that are hardcoded in the adapter code -->
                         <topic_name>Square</topic_name>
-                        <datawriter_qos base_name="BuiltinQosLib::Generic.Common"/>
+                        <datawriter_qos base_name="BuiltinQosLib::Generic.Common" />
                     </dds_output>
                 </route>
             </session>
         </domain_route>
     </routing_service>
 
-</dds>
+    <routing_service name="DDSToSocketAdapter">
+        <domain_route name="SocketBridge">
+            <connection name="SocketConnection" plugin_name="AdapterLib::SocketAdapter">
+                <registered_type name="ShapeType" type_name="ShapeType" />
```

Comment:
Can you check if `register_type` is now the official tag to register a type? I believe `registered_type` may be deprecated. We don't want to use deprecated tags in examples.